<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Partija</class>
 <widget class="QWidget" name="Partija">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <property name="styleSheet">
       <string notr="true">background-image: url(:/resources/resources/img/postavljanjePozadina.png);
QPushButton
{
	background:transparent;
	font: 16pt &quot;Ubuntu&quot;;
	background-color: rgb(119, 118, 123);
	color: rgb(36, 31, 49);
}</string>
      </property>
      <widget class="QPushButton" name="pushButtonZapocni">
       <property name="geometry">
        <rect>
         <x>640</x>
         <y>550</y>
         <width>290</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <pointsize>16</pointsize>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
         <stylestrategy>PreferDefault</stylestrategy>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background:transparent;
font: 16pt &quot;Ubuntu&quot;;
background-color: rgb(119, 118, 123);
color: rgb(36, 31, 49);</string>
       </property>
       <property name="text">
        <string>Započni partiju</string>
       </property>
      </widget>
      <widget class="QGraphicsView" name="graphicsViewFabrika">
       <property name="geometry">
        <rect>
         <x>640</x>
         <y>150</y>
         <width>290</width>
         <height>361</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:/resources/resources/img/postavljanjeTabla.png);</string>
       </property>
      </widget>
      <widget class="QGraphicsView" name="graphicsViewPostavka">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>150</y>
         <width>444</width>
         <height>444</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:/resources/resources/img/postavljanjeTabla.png);</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelNaslov">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>-10</y>
         <width>201</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Lohit Gujarati</family>
         <pointsize>23</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: transparent;
background-image: url(:/resources/resources/img/battleshipsNaslov.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>:/resources/resources/battleshipsNaslov.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonRandom">
       <property name="geometry">
        <rect>
         <x>640</x>
         <y>520</y>
         <width>290</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <pointsize>11</pointsize>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
         <stylestrategy>PreferDefault</stylestrategy>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background:transparent;
	font: 11pt &quot;Ubuntu&quot;;
	background-color: rgb(119, 118, 123);
	color: rgb(36, 31, 49);</string>
       </property>
       <property name="text">
        <string>Nasumično postavljanje brodova</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelKorektnaPostavka">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>640</y>
         <width>541</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background:none;
color : rgb(36, 31, 49);
background-color: Transparent;
</string>
       </property>
       <property name="text">
        <string>Brodovi moraju biti korektno postavljeni!</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonMine">
       <property name="geometry">
        <rect>
         <x>660</x>
         <y>90</y>
         <width>111</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>dodaj mine</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButtonTajmer">
       <property name="geometry">
        <rect>
         <x>800</x>
         <y>100</y>
         <width>121</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Ukljuci tajmer</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2">
      <property name="styleSheet">
       <string notr="true">background-image: url(:/resources/resources/img/art.jpg);</string>
      </property>
      <widget class="QTableWidget" name="tableWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>401</width>
         <height>401</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>401</width>
         <height>401</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>404</width>
         <height>404</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background:none;
background-image: url(:/resources/resources/img/doodle.png);</string>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>40</number>
       </attribute>
       <attribute name="horizontalHeaderHighlightSections">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderHighlightSections">
        <bool>false</bool>
       </attribute>
       <row>
        <property name="text">
         <string>New Row</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>New Row</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>New Row</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>New Row</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>New Row</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>New Row</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>New Row</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>New Row</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>New Row</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>New Row</string>
        </property>
       </row>
       <column>
        <property name="text">
         <string>New Column</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>New Column</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>New Column</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>New Column</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>New Column</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>New Column</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>New Column</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>New Column</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>New Column</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>New Column</string>
        </property>
       </column>
       <item row="0" column="0">
        <property name="text">
         <string/>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
        </property>
       </item>
       <item row="0" column="1">
        <property name="flags">
         <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
        </property>
       </item>
      </widget>
      <widget class="QTableWidget" name="tableWidget_2">
       <property name="geometry">
        <rect>
         <x>580</x>
         <y>70</y>
         <width>401</width>
         <height>401</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>400</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>402</width>
         <height>402</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background:none;
background-image: url(:/resources/resources/img/doodle.png);</string>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="autoScroll">
        <bool>false</bool>
       </property>
       <property name="showGrid">
        <bool>true</bool>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>40</number>
       </attribute>
       <attribute name="horizontalHeaderHighlightSections">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderHighlightSections">
        <bool>false</bool>
       </attribute>
       <row>
        <property name="text">
         <string>New Row</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>New Row</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>New Row</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>New Row</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>New Row</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>New Row</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>New Row</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>New Row</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>New Row</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>New Row</string>
        </property>
       </row>
       <column>
        <property name="text">
         <string>New Column</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>New Column</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>New Column</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>New Column</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>New Column</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>New Column</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>New Column</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>New Column</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>New Column</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>New Column</string>
        </property>
       </column>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>630</y>
         <width>551</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 300 italic 21pt &quot;Ubuntu&quot;;
background-color: transparent;
border: 1px solid #000000;
border-radius: 5px;
</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; color:#ffffff;&quot;&gt;Tvoj potez&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="margin">
        <number>1</number>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonOdustani">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>660</y>
         <width>176</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:transparent;
font: 16pt &quot;Ubuntu&quot;;
background-color: rgb(26, 95, 180);
color: rgb(255, 255, 255);
</string>
       </property>
       <property name="text">
        <string>Odustani</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelPobednik">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>600</y>
         <width>611</width>
         <height>121</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:transparent;
font: 700 33pt &quot;Ubuntu&quot;;

</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>206</y>
         <width>71</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:transparent;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="brod0">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>420</y>
         <width>80</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:transparent;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="brod2">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>230</y>
         <width>120</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:transparent;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="brod4">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>340</y>
         <width>200</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:transparent;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="brod1">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>230</y>
         <width>80</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:transparent;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="brod3">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>290</y>
         <width>160</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:transparent;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="brod0_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>40</width>
         <height>80</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:transparent;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="brod1_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>40</width>
         <height>80</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:transparent;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="brod2_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>40</width>
         <height>120</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:transparent;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="brod3_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>40</width>
         <height>160</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:transparent;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="brod4_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>40</width>
         <height>200</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:transparent;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>900</x>
         <y>670</y>
         <width>61</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:transparent;
background-color: rgb(26, 95, 180);
font: 17pt &quot;Ubuntu&quot;;
color:rgb(255, 255, 255);
</string>
       </property>
       <property name="text">
        <string>🎵</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelNaslov_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>231</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Lohit Gujarati</family>
         <pointsize>23</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background: transparent;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>:/resources/resources/battleshipsNaslov.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="labelTajmer">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>10</y>
         <width>101</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Waree</family>
         <pointsize>15</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
      <zorder>labelPobednik</zorder>
      <zorder>tableWidget</zorder>
      <zorder>tableWidget_2</zorder>
      <zorder>label</zorder>
      <zorder>pushButtonOdustani</zorder>
      <zorder>brod0</zorder>
      <zorder>brod2</zorder>
      <zorder>brod4</zorder>
      <zorder>brod1</zorder>
      <zorder>brod3</zorder>
      <zorder>brod0_2</zorder>
      <zorder>brod1_2</zorder>
      <zorder>brod2_2</zorder>
      <zorder>brod3_2</zorder>
      <zorder>brod4_2</zorder>
      <zorder>pushButton</zorder>
      <zorder>labelNaslov_2</zorder>
      <zorder>label_2</zorder>
      <zorder>labelTajmer</zorder>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
